doc/manual/manual.tex

```diff
@@ -1486,6 +1486,85 @@ \subsubsection{Free surface stabilization}
 
 An example of a simple model which uses a free surface may be found in Section \ref{sec:cookbooks-freesurface}.
 
+
+\subsection{Nullspace removal}
+
+The Stokes equation (\ref{eq:stokes-1}) only involves symmetric gradients of the velocity, and as such 
+the velocity is determined only up to rigid-body motions (that is to say, translations and rotations).
```

Comment:
...unless you specify velocity boundary conditions that determine the velocity uniquely.